uwsgi-sogs.ini

```diff
@@ -0,0 +1,33 @@
+# uwsgi configuration for listening directly on port 80; this is the simplest setup, but is less
```

Comment:
stray config file checked in

Reply:
you mean that this file shouldn't be commited right?